src/components/single-product-page/ProductDescription.css

```diff
@@ -0,0 +1,13 @@
+.description-items {
+  list-style: decimal;
+  margin-left: 1rem;
```

Comment:
Appreciation : I see that everywhere you have used rems and you have not missed a single time and used px.